pks-pipelines/minimum-release-verification/upload-kubo-release/task.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eux pipefail
+
+source git-kubo-ci/pks-pipelines/minimum-release-verification/utils/all-env.sh
+
+pushd git-pks-kubo-release
```

Comment:
did you mention that the release would have the git sha as version?  that sounds like an interesting fact to call out in a comment somewhere...